src/envattrs/__init__.py

```diff
@@ -27,15 +27,16 @@ def _iter(cls, prefix, source):
 
     for field in fields(cls):
         sub = field.metadata.get(SubAttrs, None)
-        key = f"{prefix.upper()}_{field.name.upper()}"
+        field_name = field.name.strip("_")
```

Comment:
I think this should only [`lstrip`](https://github.com/python-attrs/attrs/blob/ccecbf3ef7b56d53cd8d3a52c72e07a488d825d6/src/attr/_make.py#L2168)

Reply:
you are absolutely right